### DISCUSSION TITLE
Puan dağılımı

### DISCUSSION THREAD

**Discussion by nns95** [IBAX-io/go-ibax#121] [Q&A] [answered]
Puan dağılımı sürecinde kilitler açıldığında genellikle puanların sıfır gözükmesi bir sorun yaşatır mi ya da tweeter hesabımizlla sürekli bağlantinin kesilmesi sorun yaşatır mj

**Comment by byblckd** [comment, marked as answer]
> Puan dağılımı sürecinde kilitler açıldığında genellikle puanların sıfır gözükmesi bir sorun yaşatır mi ya da tweeter hesabımizlla sürekli bağlantinin kesilmesi sorun yaşatır mj
ibax puanlar için twitter bağlantısı bulunmakta ve kazanılan puanlarınız sisteme otomatik işlenip kayıt ettğiniz ibax cüzdanına otomatik bağlantı içinde. yani twitter bağnatınızın kopması kazandığınız puanların kaybedilmesine sebep olmaz önemli olan cüzdanınızın kayıtlı olmasıdır

**Reply by nns95** [reply]
Teşekkürler

**Reply by byblckd** [reply]
sadece dikkat etmeniz gereken twitter görevlerini yaparken yani retweet yaparken twitter hesabınızın ibax a bağlı  olduğundan emin olun :)
Rica ederim iyi günler

**Comment by cartvecurt** [comment]
Olacağını sanmıyorum ama olursa bile mail yoluyla ulaşırsan bu twitterdan bu cüzdanla yaptım şeklinde sorununu çözerler bence

**Reply by nns95** [reply]
Teşekkürler

**Reply by alisabanoglu** [reply]
Teşekkürler

**Reply by cartvecurt** [reply]
Rica ederim

**Reply by ates315** [reply]
teşekkürler

**Comment by rocknstorm** [comment]
Sonuçta cüzdanımız şifremiz filan her şey belli. Düzeltilebilecek bir durum diye düşünüyorum.

**Comment by hemresenol** [comment]
Hayır cüzdan ağında herşey kayıtlı bir sıkıntı olmaz 👍
